tests/backend/test_datadoc_metadata.py

```diff
@@ -262,6 +263,46 @@ def test_dataset_status_default_value(
     assert expected_type == datadoc_metadata.meta.dataset.dataset_status
 
 
+@pytest.mark.parametrize(
```

Comment:
We should add some more cases to this test, to cover all the possible values of `Assessment`.